ui/app/components/license-banners.js

```diff
@@ -54,9 +58,12 @@ export default class LicenseBanners extends Component {
   @action
   dismissBanner(dismissAction) {
     // if a client's version changed their old localStorage key will still exists.
-    localStorage.cleanUpStorage('dismiss-license-banner', `dismiss-license-banner-${this.currentVersion}`);
+    localStorage.cleanUpStorage(
+      'dismiss-license-banner',
+      `dismiss-license-banner-${this.currentVersion}-${this.args.expiry}`
```

Comment:
I like the idea of using the expiry as the identifier! Nice job working with what's available and not revealing sensitive information. Seeing this updated in multiple places, I'm wondering if it'd be helpful to have a getter for the storage key to minimize the room for error (i.e. typos)n in the future? 

```js
get dismissedBannerKey (){
 return `dismiss-license-banner-${this.currentVersion}-${this.args.expiry}`
}

Reply:
Great idea!